Provider/build.gradle.kts

```diff
@@ -55,6 +55,7 @@ dependencies {
     implementation(
         "org.jetbrains.kotlinx:kotlinx-serialization-json:${Versions.kotlinxSerialization}"
     )
+    implementation("androidx.lifecycle:lifecycle-process:2.6.1")
```

Comment:
Thinking out loud:
Now this may be a library that already would exist in the host app, but I feel we need some way to measure binary increase for this type of changes.
Ideally we would ship "integrations/add-ons" like this in a separate artifact, like `confidence-analytics-lifecycle`.

Reply:
yeah for sure we should separate the producers.

Reply:
created a ticket [here](https://spotify.atlassian.net/browse/HAW-539)